components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml

```diff
@@ -41,6 +41,7 @@
 
 {{velocity}}
 $xwiki.ssx.use('PhenoTips.PatientSheetCode', {'colorTheme' : "$xwiki.getSpacePreference('colorTheme')"})##
+$xwiki.ssx.use('PhenoTips.FamilyRecordField_pedigree')##
```

Comment:
This should be in `PhenoTips.FamilyRecordField_pedigree`, since it only applies to content generated by that extension. That is, if the SSX survives the review.